packages/react-core/src/deprecated/components/ApplicationLauncher/examples/ApplicationLauncher.md

```diff
@@ -13,7 +13,7 @@ import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
 import { Link } from '@reach/router';
 import pfLogoSm from './pf-logo-small.svg';
 
-Note: Application launcher is built on Dropdown, for extended API go to [Dropdown](/documentation/react/components/dropdown) documentation.
+Note: Application launcher is built on Dropdown, for extended API go to [Dropdown](/documentation/react/components/menus/dropdown) documentation.
```

Comment:
The links in this paragraph still don't work. For some reason they are in the form "/documentation/react/components..." Shouldn't they just be "/Components/..."?